app/views/vacancies/_schools.js.haml

```diff
@@ -0,0 +1,57 @@
+:javascript
+  const getSchools = () => {
+    if (document.getElementById('map').dataset.schools) {
+      return JSON.parse(document.getElementById('map').dataset.schools);
+    }
+  };
+
+  // Stops working if I change to const initMap = () => {}
+  function initMap() {
+    const schools = getSchools();
+
+    const bounds = new google.maps.LatLngBounds();
+
+    // Create map with default position and zoom as fitBounds sometimes
+    // does nothing
+    
+    const school1 = schools[0]
+    const firstLatLng = { lat: school1.lat, lng: school1.lng }
+
+    const map = new google.maps.Map(document.getElementById('map'), {
+      zoom: 16,
+      center: firstLatLng
+    })
+
+    // Use one infoWindow for all markers.
+    // This means that only one infoWindow can be open at a time.
+    const infoWindow = new google.maps.InfoWindow;
+    
+    schools.forEach((school) => {
+      const contentString = `<p>${school.name_link}</p>` +
+                            `<p>${school.address}</p>` +
+                            `<p>${school.school_type}</p>`
+
+      const latLng = { lat: school.lat, lng: school.lng };
```

Comment:
these shouldnt really be const's as they are in iterative array, bit not a huge issue at all

Reply:
So `let`?

Reply:
actually forget it consts are fine